index.bs

```diff
@@ -2153,6 +2158,82 @@ It is recommended that once [=explicit consent=] is granted for a specific [=/or
 
 Regardless of how long the user agent chooses to persist the user's consent, [=sensitive information=] MUST only be exposed by an {{XRSession}} which has not [=ended=]. 
 
+Data adjustments {#data-adjustments-header}
+----------------
+
+In some cases, security and privacy threats can be mitigated through <dfn>data adjustment</dfn>s such as throttling, quantizing, rounding, limiting, or otherwise manipulating the data reported from the [=/XR device=]. This may sometimes be necessary to avoid fingerprinting, even in situations when [=user intent=] has been established.  However, [=data adjustment=] mitigations MUST only be used in situations which would not result in user discomfort.
+
+### Throttling ### {#throttling}
+Throttling is when [=sensitive information=] is reported at a lower frequency than otherwise possible. This mitigation has the potential to reduce a site's ability to infer user intent, location, or perform user profiling. However, when not used appropriately throttling runs a significant risk of causing user discomfort. In addition, under many circumstances it may be inadequate to provide a complete mitigation.
```

Comment:
nit: Grammar is a bit wrong in this sentence.  "location" probably needs to be prefaced with "infer"

>This mitigation has the potential to reduce a site's ability to infer user intent, location, or perform user profiling.